src/sentry/static/sentry/app/views/eventsV2/table/columnEditModal.tsx

```diff
@@ -60,8 +61,13 @@ class ColumnEditModal extends React.Component<Props, State> {
         </Header>
         <Body>
           <Instruction>
-            {t(
-              'To stack events, add functions that may take in additional parameters. Tag and field columns will help you view more details about the events.'
+            {tct(
+              'To stack events, add [functionLink: functions] f(x) that may take in additional parameters. [tagLink: Tags] and [fieldLink: fields] columns will help you view more details about the events (i.e. title).',
+              {
+                functionLink: <Link to="/" />,
+                tagLink: <Link to="/" />,
+                fieldLink: <Link to="/" />,
```

Comment:
Where are these links going to go?

Reply:
LOL sorry you responded faster than my second commit. Fixed!